src/core/state/contacts/index.ts

```diff
@@ -10,6 +10,7 @@ export interface Contact {
 
 export interface ContactsStore {
   contacts: { [address: Address]: Contact };
+  selectedContactAddress: Address | null;
```

Comment:
instead of holding this in a store what do you think of using a search param in the page? 
like navigating to `${ROUTES.send}?to=${contactAddress}`

Reply:
Yeah that'd be a better approach, but since i saw we use states all over the place i choose to stick with that approach to not complicate things with states and queries. [Here](https://github.com/rainbow-me/browser-extension/blob/master/src/entries/popup/pages/send/index.tsx#L490-L501) is an example.

Reply:
If there's a better approach, do the better approach @magiziz. We do not need to copy things from other portions of the application if there's a better way.

Reply:
Yep. Queries should be much easier in this. Changes are pushed 🙏